scripts/lib/CIME/case_st_archive.py

```diff
@@ -247,6 +327,14 @@ def _archive_restarts(case, archive, archive_entry,
     # copy latest restart files to archive restart directory
     histfiles_savein_rundir = []
 
+    # determine function to use for last set of restart files
```

Comment:
This allows us to link rather than copy the restarts in erp and similar tests. This saves some time and disk space, but I'm not tied to this if others feel it adds too much complexity.